### ISSUE TITLE
Variable definitions need to default to 0

### ISSUE THREAD

**Issue by nickwanninger** [nickwanninger/geode#5]
In gitlab by @nickwanninger on Jun 24, 2018, 23:54

`int a;` has random values. If a variableDefn has no body, ensure it is set to 0 and type cast correctly.

**Comment by nickwanninger** [comment]
In gitlab by @nickwanninger on Jun 24, 2018, 23:54

This code can be used to test:
```
func main(int argc) byte {
	int a;
	printf("%d\n", a);
	return 0;
}
```

**Comment by nickwanninger** [comment]
In gitlab by @nickwanninger on Jun 24, 2018, 23:57

fixed in 5c09e0f61536724695a75e3e70566f02060de922

**Comment by nickwanninger** [comment]
In gitlab by @nickwanninger on Jun 24, 2018, 23:57

closed
